substrate/frame/revive/src/exec.rs

```diff
@@ -282,6 +282,14 @@ pub trait PrecompileExt: sealing::Sealed {
 		self.gas_meter_mut().charge(crate::RuntimeCosts::Precompile(weight))
 	}
 
+	fn adjust_gas(
+		&mut self,
+		charged: crate::gas::ChargedAmount,
+		actual_costs: crate::RuntimeCosts,
+	) {
+		self.gas_meter_mut().adjust_gas(charged, actual_costs);
+	}
+
```

Comment:
we should do like in `charge` 
since you are always going to use the RuntimeCosts::Precompile here, you are better of just passing the actual_weight, and then wrapping that with RuntimeCosts::Precompile

Reply:
Done